packages/config/config/devices/0x0086/dsc11.json

```diff
@@ -17,329 +18,306 @@
+		},
+		"103[0x4000]": {
+			"$import": "templates/aeotec_template.json#auto_report_group3_amp",
+			"label": "Automatic Report: Group 3 - Power (Socket 6)"
+		},
 		"100": {
-			"label": "Reset 0x65~0x67 to default value",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 0,
-			"unsigned": true,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
+			"$import": "templates/aeotec_template.json#reset_parameters",
+			"label": "Reset Parameters 101-103 to Default Values"
 		},
 		"111": {
-			"label": "Interval to send out reports of Group 1",
-			"valueSize": 4,
-			"minValue": 1,
-			"maxValue": 2678400,
-			"defaultValue": 600,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
+			"$import": "templates/aeotec_template.json#auto_report_interval_group1"
 		},
 		"112": {
-			"label": "Interval to send out reports of Group 2",
-			"valueSize": 4,
-			"minValue": 1,
-			"maxValue": 2678400,
-			"defaultValue": 600,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
+			"$import": "templates/aeotec_template.json#auto_report_interval_group2"
 		},
 		"113": {
-			"label": "Interval to send out reports of Group 3",
-			"valueSize": 4,
-			"minValue": 1,
-			"maxValue": 2678400,
-			"defaultValue": 600,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
+			"$import": "templates/aeotec_template.json#auto_report_interval_group3"
 		},
-		"101[0xff]": {
-			"label": "select channels to report consumtion as item 1",
-			"description": "A binary field for reporting consumption (kWh) of channels",
-			"unit": "kWh",
-			"valueSize": 4,
-			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 0,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
+		"110": {
+			"$import": "templates/aeotec_template.json#reset_parameters",
+			"label": "Reset Parameters 111-113 to Default Values"
 		},
-		"101[0xff00]": {
-			"label": "Select channels to report power as item 1",
-			"description": "A binary field to choose channels to report power",
-			"unit": "W",
-			"valueSize": 4,
-			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 0,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
+		"252": {
+			"$import": "templates/aeotec_template.json#lock_configuration"
 		},
-		"102[0xff]": {
-			"label": "Item 2 (kWh)",
-			"description": "Channel(s) to auto-report kWh",
+		"253": {
+			"label": "Meter Calibration Mode",
 			"valueSize": 4,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 1,
 			"defaultValue": 0,
+			"unsigned": true,
 			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
+			"writeOnly": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Normal Operation",
+					"value": 0
+				},
+				{
+					"label": "Enter Meter Calibration Mode",
+					"value": 1
+				}
+			]
```

Comment:
We can. I believe you set this and then it exits the calibration mode after a period of time. But this is write only so it would continue to read Enabled. Your call.